runtime/doc/api.txt

```diff
@@ -3108,6 +3108,204 @@ nvim_tabpage_set_var({tabpage}, {name}, {value})
+
+                    -- Vimscript function name (as a string)
+                    local myvimfun = "g:MyVimFunction"
+
+                    vim.api.nvim_create_autocmd({
+                      event = {"BufEnter", "BufWinEnter"},
+                      pattern = {"*.c", "*.h"},
+                      callback = myluafun,  -- Or myvimfun
+                    })
+<
+
+                Example using command: >
+                    vim.api.nvim_create_autocmd({
+                      event = {"BufEnter", "BufWinEnter"},
+                      pattern = {"*.c", "*.h"},
+                      command = "echo 'Entering a C or C++ file'",
+                    })
+<
+
+                Example values for patterns: >
+                  pattern = "*.py,*.pyi"
+                  pattern = "*.py"
+                  pattern = {"*.py"}
+                  pattern = { "*.py", "*.pyi" }
+                  -- NOT supported
+                  pattern = {"*.py,*.pyi"}
+<
+
+                Examples values for events: >
+                  event = "FileType,CursorHold"
+                  event = "BufPreWrite"
+                  event = {"BufPostWrite"}
+                  event = {"CursorHold", "BufPreWrite", "BufPostWrite"}
+<
+
+                Parameters: ~
+                    {opts}  Dictionary of autocommand options:
+                            • group: (string) optional, the group name
+                              |autocmd-groups|.
+                            • event: (string|array) required, event or events
+                              to match against |autocmd-events|.
+                            • pattern: (string|array) optional, pattern or
+                              patterns to match against |autocmd-pattern|.
+                            • buffer: (integer) optional, buffer number for
+                              buffer local autocmds (can't be used together
+                              with pattern) |autocmd-buflocal|.
+                            • desc: (string) optional, description of the
+                              autocmd.
+                            • callback: (function|string) optional, Lua
+                              function or Vim function (as string) to execute
+                              on event.
+                            • command (string) optional, Vim command to
+                              execute on event (can't be used together with
+                              callback).
+                            • once (boolean) optional, defaults to false. Run
+                              the autocmd only once |autocmd-once|.
+                            • nested: (boolean) optional, defaults to false.
+                              Run nested autocmds |autocmd-nested|.
+
+                Return: ~
+                    The id of the created autocommand.
+
+                See also: ~
+                    autocmd
+                    nvim_del_autocmd()
```

Comment:
It looks like the `@see` links are not automatically made into links, so all of the `@see foo` lines in the docstrings should be changed to `@see |foo|`.

Reply:
Done